lib/features/family/features/reflect/presentation/pages/gather_around_screen.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/features/family/app/injection.dart';
+import 'package:givt_app/features/family/extensions/extensions.dart';
+import 'package:givt_app/features/family/features/background_audio/bloc/background_audio_cubit.dart';
+import 'package:givt_app/features/family/features/background_audio/presentation/fun_background_audio_widget.dart';
+import 'package:givt_app/features/family/features/reflect/presentation/pages/summary_screen.dart';
+import 'package:givt_app/features/family/shared/design/components/components.dart';
+import 'package:givt_app/features/family/shared/widgets/texts/shared_texts.dart';
+import 'package:givt_app/features/family/utils/utils.dart';
+import 'package:givt_app/shared/models/analytics_event.dart';
+import 'package:givt_app/shared/widgets/fun_scaffold.dart';
+
+class GatherAroundScreen extends StatefulWidget {
+  const GatherAroundScreen({super.key});
+
+  @override
+  State<GatherAroundScreen> createState() => _GatherAroundScreenState();
+}
+
+class _GatherAroundScreenState extends State<GatherAroundScreen> {
+  bool _hasPlayedAudio = false;
+  bool isFirstRoundofFirstGame = true;
```

Comment:
When you reach this screen it will never be the first "round" of the first game (unless people break off the game early) because this screen is shown at the very end of the game right? Even in a 2-player game, if you fully play it this screen will be at the 2nd/last round of the first game.

Reply:
Hmm thats true indeed.<br>It was to prevent users from skipping the screen, but maybe it's fine to do that? :)<br>Or should it be based on first game?

Reply:
Yea I think just purely first game instead of first round of first game is fine!